vtStorManaged.Test/obj/x86/Release/vtStorManaged.Test.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,7 @@
+D:\Works\vtStorX\vtStorInternal\vtStor\vtStorManaged.Test\bin\x86\Release\vtStorManaged.Test.dll
```

Comment:
Configure the output directory and "intermediate" directory properly and add these to git ignore.